lib/rialto/etl/writers/json_list_writer.rb

```diff
@@ -4,10 +4,10 @@ module Rialto
   module Etl
     # Holds writers for use in Traject mappings
     module Writers
-      # Write JSON-LD records. This writer conforms to Traject's
```

Comment:
Changes to this file reflect that there's basically nothing in the writer that is JSON-LDish. Its purpose is to write out a list of JSON objects so that we can take Traject's output and stuff it in a file that is valid JSON.